src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -1397,6 +1397,8 @@ class BookingService(
       return fieldValidationError
     }
 
+    val findExistingDeparturesForBooking = departureRepository.findAllDeparturesByBookingId(booking.id)
```

Comment:
This call to `DepartureRepository` is unnecessary and can be replaced with something like:

```suggestion
    val isFirstDeparture = booking.departures.isNullOrEmpty()
```

Reply:
I tried the above approach and didnt work, let me know if you can make it work